en/guides/user-management/README.md

```diff
@@ -0,0 +1,127 @@
+---
+title: User management
+description: "Tutorial: user management"
+---
+
+# User management
+
+All **users** can [sign in](/en/for-students#sign-in) and manage their [user profile](/en/for-students#user-profile). Some actions on Dodona require more [permissions](#permissions). For this reason, we use three roles: [students](#student), [teachers](#teacher) and [admins](#admin). These roles can be [assigned](#assign-roles) to users, for instance a teacher can appoint another teacher.
+
+::: tip
+
+You can see that you are a [teacher](#teacher) when you can see the **management menu** (menu <span class="guilabel">Admin</span>) left of the [user menu](/en/for-students#user-menu) at the right side of the [navigation drawer](/en/for-students#navigation-drawer).
+
+![image](./staff.admin_menu.en.png)
+:::
+
+::: tip Important
+
+We divide the [enrolled](/en/for-students#course-register) members of a [course](/en/course-management#course) in [students](#student) and
+[course managers](/en/course-management#course-manager) depending on the [management permissions](#management-permissions)they have been granted for the course. Only course managers can manage a course. The 
+[managing](/en/course-management#managing-course-users) of 
+[course users](/en/course-management#course-user) and [appointing](/en/course-management#appointing-course-managers) course managers is discussed in [course management](/en/course-management).
+:::
+
+## Navigating to a user
+
+As a [teacher](#teacher), you navigate to the [user overviewt](#user-overview) by selecting <span class="guilabel">Users</span> in the [management menu](#management-menu) at the right side of the [navigation panel](/en/for-students#navigation-panel).
+
+![admin menu users](./staff.admin_menu_users.en.png)
+
+## User overview
```

Comment:
The screenshots and documentation in this section are using the course wide users table. I think this section needs a bit of a rewrite to reflect the current situation.

Teachers should now search for users within their course page since the general users table has become admin-only.